config.yaml

```diff
@@ -22,14 +22,19 @@ logging:
 
 # Statsd metrics reporter
 metrics:
+  enable_prometheus: true
```

Comment:
This is not really needed, we can just check if prometheus is explicitly configured.